### PR TITLE
[DARGA] Indicate duplicate Shopping cart items

### PR DESCRIPTION
**Pull request by himdel**
This is a darga version of #199.

Mostly identical to the euwe version, except we're using `Notifications` instead of `EventNotifications`.

darga: https://bugzilla.redhat.com/show_bug.cgi?id=1380170

### CODE REVIEW

File: client/app/states/marketplace/details/details.state.js [hunk 1]
```diff
@@ -84,21 +85,34 @@
...
-      .then(addSuccess, addFailure);
+      .then(addSuccess, addFailure)
+      .then(function() {
+        vm.addingToCart = false;
+      });
 
       function addSuccess(result) {
-        Notifications.success(__("Item added to shopping cart"));
+        if (result.duplicate) {
+          Notifications.success(__("Item added to shopping cart, but it's a duplicate of an existing item"));
```
Comment (AllenBW) [inline client/app/states/marketplace/details/details.state.js]: Heyo finally found something!!  `EventNotification` needs some lovin here

File: client/app/states/marketplace/details/details.state.js [hunk 2]
```diff
@@ -84,21 +85,34 @@
...
+        } else {
+          Notifications.success(__("Item added to shopping cart"));
```
Comment (AllenBW) [inline client/app/states/marketplace/details/details.state.js]: and here

File: client/app/states/marketplace/details/details.state.js (hunk 1 above)
Comment (himdel) [inline client/app/states/marketplace/details/details.state.js]: @AllenBW nope, this is a darga backport of #199. We don't have EventNoitification in darga ;)

File: client/app/states/marketplace/details/details.state.js (hunk 2 above)
Comment (himdel) [inline client/app/states/marketplace/details/details.state.js]: [Sorry](https://github.com/ManageIQ/manageiq-ui-self_service/pull/214#discussion_r82041234) ;)

File: client/app/states/marketplace/details/details.state.js (hunk 1 above)
Comment (AllenBW) [inline client/app/states/marketplace/details/details.state.js]: @himdel SNAP!! awww, now I feel bad that I couldn't contribute.. WAY TO MAKE A FLAWLESS PR 🤕 🤒

**Pull request merged**